.github/workflows/main.yml

```diff
@@ -94,9 +109,31 @@ jobs:
           # XCFrameworks are forwards-compatible but not backwards-compatible.
           # The Xcode version we use for this job is that oldest Xcode version that
           # will be able to use these XCFrameworks and the lottie-spm package.
-          xcode: '13.2.1' # Swift 5.5.2
+          xcode: '13.4.1' # Swift 5.6.1
```

Comment:
Bumped the XCFramework to be built by Xcode 13.4.1, the last version with Swift 5.6 (since we dropped support for Swift 5.5)